atspi-proxies/Cargo.toml

```diff
@@ -9,7 +9,7 @@ homepage = "https://github.com/odilia-app/atspi"
 keywords = ["screen-reader", "accessibility", "a11y", "tts", "linux"]
 categories = ["accessibility", "api-bindings"]
 edition = "2021"
-rust-version = "1.68.2"
+rust-version.workspace = true
```

Comment:
Although this is valid... It breaks the `cargo msrv verify` command. Specifying a crate like `atspi` (the crate, not the workspace), or attempting to run it in the workspace: neither work.

Although I agree we should probably just have a workspace MSRV, I'm wondering if there's a better verification tool that will handle the `.workspace` suffix?

Reply:
> I'm wondering if there's a better verification tool that will handle the `.workspace` suffix?

If it makes our lives easier, I'll happily revert this change.

Reply:
After further explanation from DataTriny, this looks like an easily solvable problem.